easy-random-core/src/main/java/org/jeasy/random/util/ReflectionUtils.java

```diff
@@ -551,6 +537,16 @@ private static void rejectUnsupportedTypes(Class<?> type) {
         }
     }
 
+    /**
+     * Get the write method for given field.
+     *
+     * @param field field to get the write method for
+     * @return Optional of write method or empty if field has no write method
+     */
+    private static Optional<Method> getWriteMethod(Field field) {
```

Comment:
This could public to be consistent with `getReadMethod` which is public.  I already fixed that on merge.